pkg/operator/ceph/object/user.go

```diff
@@ -183,6 +183,7 @@ func DeleteUser(c *Context, id string, opts ...string) (string, error) {
 	return result, errors.Wrap(err, "failed to delete s3 user")
 }
 
+// SetQuotaUserBucketMax will set maximum bucket quota for a user
```

Comment:
Can you explain why `--quota-scope=user` is being used? In my (brief) reading it seems we should be using the _bucket_ scope. Eg. create quota on bucket owned by the uid of the owner of the bucket provisioned via the obc. It seems this would allow quota to apply to additional (brownfield) access to that bucket.

Reply:
> Can you explain why `--quota-scope=user` is being used? In my (brief) reading it seems we should be using the _bucket_ scope. Eg. create quota on bucket owned by the uid of the owner of the bucket provisioned via the obc. It seems this would allow quota to apply to additional (brownfield) access to that bucket

 If it bucket quota I thought brown field cases cannot be handled properly. As discussed before multiple obc can have bucket, so setting user scope make sense for me

Reply:
OK, I understand this reasoning. But this means there is no way within Kubernetes to set a bucket quota -- just user quotas. Is this acceptable?

Reply:
@jeffvance IMO bucket quota can be applicable to `CephObjecrUser` CRD's. I didn't work anything wrt that till now